Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.4.5"
 Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+StringEncodings = "69024149-9ee7-55f6-a4c4-859efe599b68"
```

Comment:
Does this need a compat entry, or is it part of the StdLib?

Reply:
It definitely needs a compat entry.